diffusion/callbacks/log_diffusion_images.py

```diff
@@ -121,8 +121,9 @@ def __init__(self,
                 clip_attention_mask = clip_attention_mask.cpu().to(torch.long)
 
                 latent_batch['T5_LATENTS'] = t5_latents
+                latent_batch['T5_ATTENTION_MASK'] = t5_attention_mask
```

Comment:
ideally these key names wouldn't be hardcoded and would be grabbed from the dataset class but i think this OK for now

Reply:
Changed these to come from arguments to the model class, but didn't yet make the change to grab them from the dataset class.